README.md

```diff
@@ -3,14 +3,17 @@
 	* /README.md and docs/README.md should be identical copies (symlinks don't work)
 	* Use only *absolute links* in these files. Relative links will break.
 !-->
+
 [![docs on_gitbook](https://img.shields.io/badge/docs-on_gitbook-blue.svg?style=flat-square)](https://quiltdocs.gitbook.io/t4/)
 [![chat on_slack](https://img.shields.io/badge/chat-on_slack-blue.svg?style=flat-square)](https://slack.quiltdata.com/)
 [![codecov](https://codecov.io/gh/quiltdata/t4/branch/master/graph/badge.svg)](https://codecov.io/gh/quiltdata/t4)
 [![pypi](https://img.shields.io/pypi/v/t4.svg?style=flat-square)](https://pypi.org/project/t4/)
 
-
 ## Overview
-[Rethinking S3: Announcing T4, a team data hub](https://blog.quiltdata.com/rethinking-s3-announcing-t4-a-team-data-hub-8e63ce7ec988).
+
+Quilt 3 (working title Quilt T4) is a data collaboration platform for S3.
```

Comment:
I would say `(formerly T4)` or drop altogether. Per Marketing we don't have plans to use "t4" moving forward.